models/public/background-matting-mobilenetv2/accuracy-check.yml

```
@@ -0,0 +1,54 @@
+models:
+  - name: background-matting-mobilenetv2
+
+    launchers:
+      - framework: openvino
+        adapter:
+          type: background_matting_with_pha_and_fgr
+          alpha_out: pha
+          foreground_out: fgr
+        inputs:
+          - name: src
+            type: INPUT
+            value: com*
+          - name: bgr
+            type: INPUT
+            value: bgr*
+
+    datasets:
+      - name: HumanMattingClips120
+        reader: pillow_imread
+
+        preprocessing:
+          - type: rgb_to_bgr
+          - type: resize
+            dst_width: 1280
+            dst_height: 720
+            use_pillow: True
+
+        metrics:
+          - name: alpha_MAD
+            type: mad
+            prediction_source: pha
+            process_type: alpha
+            reference: 4.32
```

According to reference for converted model
```suggestion
            reference: 4.35
```